src/components/BaseTextInput/index.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import {
+    // eslint-disable-next-line no-restricted-imports
+    TextInput as RNTextInput,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+
+const propTypes = {
+    /** A ref to forward to the text input */
+    forwardedRef: PropTypes.func,
+};
+
+const defaultProps = {
+    forwardedRef: null,
```

Comment:
Type does not match with the default value. 
```suggestion
    forwardedRef: PropTypes.func,
};

const defaultProps = {
    forwardedRef: () => {},
```

Reply:
I flow the same behavior 

https://github.com/Expensify/App/blob/0368dad861cf0c28146ac8a8f8646d8d96535270/src/components/TextInputFocusable/index.ios.js#L46

Reply:
That is a bad example of old cold. The correct one would be the suggested changes. Please do this change.